### PR TITLE
Unique and better naming for the apps

### PR DESCRIPTION
**Pull request by 7opf**

### CODE REVIEW

File: identity-enabler/deviceId-mobile-app/android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
         <activity
             android:configChanges="orientation|keyboardHidden|keyboard|screenSize|locale|smallestScreenSize|screenLayout|uiMode"
-            android:name="org.iota.zebra.MainActivity"
+            android:name="org.iota.zebra.device.MainActivity"
```
Comment (jmcanterafonseca-iota) [inline identity-enabler/deviceId-mobile-app/android/app/src/main/AndroidManifest.xml:17]:
```suggestion
            android:name=".MainActivity"
```

File: identity-enabler/deviceId-mobile-app/android/app/src/main/java/com/hello/svelte/MainActivity.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package org.iota.zebra;
+package org.iota.zebra.device;
```
Comment (jmcanterafonseca-iota) [inline identity-enabler/deviceId-mobile-app/android/app/src/main/java/com/hello/svelte/MainActivity.java:1]: you need to move the folder as well

Comment (jmcanterafonseca-iota) [inline identity-enabler/deviceId-mobile-app/android/app/src/main/java/com/hello/svelte/MainActivity.java:1]:
```suggestion
package org.iota.zebra.device_id;
```

File: identity-enabler/deviceId-mobile-app/android/app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <?xml version='1.0' encoding='utf-8'?>
 <resources>
-    <string name="app_name">Zebra-IOTA Identity</string>
-    <string name="title_activity_main">Zebra-IOTA Identity</string>
-    <string name="package_name">org.iota.zebra</string>
-    <string name="custom_url_scheme">org.iota.zebra</string>
+    <string name="app_name">Device ID - IOTA SDK</string>
+    <string name="title_activity_main">Device ID - IOTA SDK</string>
+    <string name="package_name">org.iota.zebra.device</string>
```
Comment (jmcanterafonseca-iota) [inline identity-enabler/deviceId-mobile-app/android/app/src/main/res/values/strings.xml:5]:
```suggestion
    <string name="package_name">org.iota.zebra.device_id</string>
```

File: identity-enabler/holder-mobile-app/android/app/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -14,7 +14,7 @@
 
         <activity
             android:configChanges="orientation|keyboardHidden|keyboard|screenSize|locale|smallestScreenSize|screenLayout|uiMode"
-            android:name="org.iota.zebra.MainActivity"
+            android:name="org.iota.zebra.holder.MainActivity"
```
Comment (jmcanterafonseca-iota) [inline identity-enabler/holder-mobile-app/android/app/src/main/AndroidManifest.xml:17]:
```suggestion
            android:name=".MainActivity"
```

File: identity-enabler/holder-mobile-app/android/app/src/main/java/com/hello/svelte/MainActivity.java [hunk 5]
```diff
@@ -1,4 +1,4 @@
-package org.iota.zebra;
+package org.iota.zebra.holder;
```
Comment (jmcanterafonseca-iota) [inline identity-enabler/holder-mobile-app/android/app/src/main/java/com/hello/svelte/MainActivity.java:1]: you need to move the file as well

File: identity-enabler/verifier-mobile-app/android/app/src/main/AndroidManifest.xml [hunk 6]
```diff
@@ -35,6 +35,7 @@
         <activity
             android:name=".verifier.MainActivity"
             android:configChanges="orientation|keyboardHidden|keyboard|screenSize|locale|smallestScreenSize|screenLayout|uiMode"
+            android:name="org.iota.zebra.verifier.MainActivity"
```
Comment (jmcanterafonseca-iota) [inline identity-enabler/verifier-mobile-app/android/app/src/main/AndroidManifest.xml:38]:
```suggestion
            android:name=".MainActivity"
```

File: identity-enabler/verifier-mobile-app/android/app/src/main/java/com/hello/svelte/MainActivity.java [hunk 7]
```diff
@@ -0,0 +1,27 @@
+package org.iota.zebra.verifier;
```
Comment (jmcanterafonseca-iota) [inline identity-enabler/verifier-mobile-app/android/app/src/main/java/com/hello/svelte/MainActivity.java:1]: be careful, this file has been previously removed! please rebase your codebase

**Review by jmcanterafonseca-iota — CHANGES_REQUESTED** [review]
We need several improvements and a code rebase

**Comment by 7opf** [comment]
Reviews have been addressed and branch rebased + lock file regenerated.

**Pull request merged**